packages/component/src/SendBox/DictationInterims.js

```diff
@@ -44,7 +45,18 @@ export default connectDictationInterims(
     dictateState,
     styleSet
   }) =>
-    (dictateState === STARTING || dictateState === DICTATING) && (
+    (dictateState === STARTING || dictateState === STOPPING) ?
```

Comment:
We need to handle `STOPPING` because the new Speech Services SDK is very slow at stopping (> 2 seconds to stop, vs. almost immediate with Bing Speech).

Without this logic, the UI looks frozen for 2 seconds.